actions/image_partition_action.go

```diff
@@ -1,3 +1,106 @@
+/*
+ImagePartition Action
+
+This action allows to create the disk image and partitions layout mapped to
+root filesystem.
+
+Yaml syntax:
+ - action: image-partition
+   imagename: image_name
+   imagesize: size
+   partitiontype: gpt
+   partitions:
+     <list of partitions>
+   mountpoints:
+     <list of mount points>
+
+Mandatory properties:
+
+- imagename -- the name of the output image generated by 'debos'.
+
+- imagesize -- generated image size in human-readable form, examples: 100MB, 1GB, etc.
+
+- partitiontype -- partition table type. Currently only 'gpt' and 'msdos'
+partition tables are supported.
+
+- partitions -- list of partitions, at least one partition is needed.
+Partition properties are described below.
+
+- mountpoints -- list of mount points for partitions.
+Properties for mount points are described below.
+
+Yaml syntax for partitions:
+
+   partitions:
+     - name: label
+	   name: partition name
+	   fs: filesystem
+	   start: offset
+	   end: offset
+	   flags: list of flags
+
+Mandatory properties:
+
+- name -- is used for referencing named partition for mount points
+configuration (below) and label the filesystem located on this partition.
+
+- fs -- filesystem type used for formatting.
+
+NB: the FAT (vfat) filesystem is named 'fat32' in configuration file.
```

Comment:
May make sense to give a list of supported filesystems ? (Though the code is rather generic)

Reply:
Have no sense at all -- it depends only on installed mkfs.* utilities on build host with current implementation ;)

Reply:
btw, wanted to ask if it have sense to use 'vfat' instead of 'fat32'. I see 'fat32' it is needed only because of parted.

Reply:
well but there is a set of filesystems that we've tested e.g. i know ext2, ext4, fat32 and btrfs should work.  some of hte other mkfs.* scripts might be there but are untested. 

For the fat stuff, yes having some aliases may make sense to make it easier, but htat's a seperate issue ;)

Reply:
mmm... still see no sense in **supported** filesystems. But **tested** filesystems are also weird.